Fix broken links

LightIsLife opened the pull request:
fixes #808
reference to Yahooweather (now obsolete) deleted
replaced Yahooweather (obsolete) with Network binding in migration example
fixed misc links

LightIsLife commented:
Guess I forgot the sign off:
Signed-off-by: Markus Lipp lipp_markus@gmail.com (github: LightIsLife)

Confectrician commented on addons/uis.md:
```diff
@@ -11,17 +11,18 @@ openHAB offers a variety of frontends for user interaction, from administrative
-| [Android App]({{base}}/addons/uis/apps/android.html) | The native Android app to access openHAB on the go. |
-| [iOS App]({{base}}/addons/uis/apps/ios.html) | The native iOS app to access openHAB on the go. |
-| [Windows 10 App]({{base}}/addons/uis/apps/windows.html) | The native Windows 10 app to access openHAB on the go. |
+| [Android App]({{base}}/apps/android.html) | The native Android app to access openHAB on the go. |
```
I think this one won't work since the source file is in the addons tree.

`/docs/apps/android.html` (without `{{base}}`) should do the trick.

Smae goes for the next two lines.

LightIsLife replied:
You are absolutely correct the links above do not work as they were. Fixed links for all three apps. For consistency, I left it with {{base}} as I tried to use this with all other links to the docs too. If this is not longer necessary or desired, let me know and I can take all of them out.

LightIsLife commented:
And a little more comment: #808 identified a few broken links, that should be all fixed. 
By reading through the files and some more searching I found more links that appears to need fixing. 
In addition, the section migration used the yahoo weather binding as examples, however, this binding is now obsolete and I replaced it with an example for the network binding
In addition, the yahoo weather binding was used as an example for which a configuration file is needed, I just deleted this example. 
Any and all comments are welcome

Confectrician commented on configuration/things.md:
```diff
@@ -42,8 +42,8 @@ through [discovery]({{base}}/concepts/discovery.html) or by manual definition in
-For some bindings (such as the [YahooWeater]({{base}}/addons/bindings/yahooweather/readme.html) binding), manual Thing definitions are required.
-Other bindings (such as the [ZWave]({{base}}/addons/bindings/zwave/readme.html) binding) currently prefer or require the discovery method.
+For some bindings, manual Thing definitions are required.
+Other bindings (such as the [ZWave](https://www.openhab.org/addons/bindings/zwave/) binding) currently prefer or require the discovery method.
```
Do we need the domain here? Not sure.

Confectrician commented on configuration/things.md:
```diff
@@ -83,7 +83,7 @@ Thing ntp:ntp:local [ hostname="de.pool.ntp.org" ]
 Looking at the first example:
 
-- the binding ID is "network" (using the [Network Binding]({{base}}/addons/bindings/network/readme.html))
+- the binding ID is "network" (using the [Network Binding](https://www.openhab.org/addons/bindings/network/)
```
Do we need the domain here? Not sure.

LightIsLife replied:
also deleted

Confectrician commented on tutorials/beginner/configuration.md:
```diff
@@ -123,7 +123,7 @@ To demonstrate that, we'll proceed and install the Zwave binding in order to add
 As mentioned before, the pure procedure to install and configure add-ons is mostly the same with other addons.**
 
-**More information about the Zwave binding and the other available bindings can be found [on the bindings page of the user manual]({{base}}/addons/bindings.html)!**
+**More information about the Zwave binding and the other available bindings can be found [on the bindings page of the user manual](https://www.openhab.org/addons/#bindings)!**
```
Do we need the domain here? Not sure.

LightIsLife commented:
@Confectrician thanks much...just wait a little for further review, please. Somehow I seem to have myself completely confused by trying out github desktop and am struggling to find the right branch/repo to actually make additional commits (seemingly I am only offered new pull requests...) anyway...will let you know when I figured it all out.

LightIsLife commented:
Guess, I maneuvered myself in a corner from which I cannot come back from. Will close this one an open a new PR....I had already fixed the comments above, but apparently it never made it into the commit....so now I am lost what was done and was not done; will start over again, sorry for that.

Confectrician commented:
I am currently in contact with @MyRaceData to give some git/github overview/introduction with a focus on editing the docs or contributing to the openHAB project.

Maybe this could help you too?

MyRaceData commented:
Yes, I did the same sort of thing but using the online editor. Thanks Jerome with getting up to speed on making the pull requests and getting them merged. When I have the process down, perhaps I can create a how to/reference type document to help others.
